plugins/modules/aci_l4l7_policy_based_redirect_destination.py

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_policy_based_redirect_destination
+short_description: Manage L4-L7 Policy Based Redirect Destinations (vns:RedirectDest and vns:L1L2RedirectDest)
+description:
+- Manage L4-L7 Policy Based Redirect Destinations
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  policy:
+    description:
+    - The name of an existing Policy Based Redirect Policy.
+    type: str
+    aliases: [ policy_name ]
+  ip:
+    description:
+    - The destination IP for redirection.
+    - Only used if I(destination_type=l3)
+    aliases: [ redirect_ip ]
+    type: str
+  additional_ip:
+    description:
+    - The Additional IP Address for the Destination.
+    - Only used if I(destination_type=l3)
+    type: str
+  logical_device:
+    description:
+    - The destination Logical Device for redirection.
+    - Only used if I(destination_type=l1/l2)
+    type: str
+    aliases: [ logical_dev ]
+  concrete_device:
+    description:
+    - The destination Concrete Device for redirection.
+    - Only used if I(destination_type=l1/l2)
+    type: str
+    aliases: [ concrete_dev ]
+  concrete_interface:
+    description:
+    - The destination Concrete Interface for redirection.
+    - Only used if I(destination_type=l1/l2)
+    type: str
+    aliases: [ concrete_intf ]
+  mac:
+    description:
+    - The destination MAC address for redirection.
+    type: str
+    aliases: [ redirect_mac ]
+  destination_name:
+    description:
+    - The name for Policy Based Redirect destination.
+    type: str
+    aliases: [ dest_name ]
+  destination_type:
+    description:
+    - The destination type.
+    type: str
+    choices: [ l1/l2, l3 ]
+    aliases: [ dest_type ]
+  pod_id:
+    description:
+    - The Pod ID to deploy Policy Based Redirect destination on.
+    - The APIC defaults to C(1) when unset during creation.
+    type: int
+  health_group:
+    description:
+    - The Health Group to bind the Policy Based Redirection Destination to.
+    - To remove an existing binding from a Health Group, submit a request with I(state=present) and no I(health_group) value.
```

Comment:
empty "" for removal else not touch?

Reply:
I changed this so that we can delete it with "" or None